components/scream/src/physics/p3/tests/p3_run_and_cmp.cpp

```diff
@@ -261,21 +259,25 @@ int main (int argc, char** argv) {
     std::cout <<
       argv[0] << " [options] baseline-filename\n"
       "Options:\n"
-      "  -g              Generate baseline file. Default False.\n"
-      "  -f              Use fortran impls instead of c++. Default False.\n"
-      "  -t <tol>        Tolerance for relative error. Default 0.\n"
-      "  -s <steps>      Number of timesteps. Default=6.\n"
-      "  -i <cols>       Number of columns. Default=6.\n"
-      "  -r <repeat>     Number of repetitions, implies timing run (generate + no I/O). Default=0.\n"
-      "  -p <predict_nc>  yes|no|both. Default=both.\n"
+      "  -g                  Generate baseline file. Default False.\n"
+      "  -f                  Use fortran impls instead of c++. Default False.\n"
+      "  -t <tol>            Tolerance for relative error. Default 0.\n"
+      "  -s <steps>          Number of timesteps. Default=6.\n"
+      "  -d <seconds>        Length of timestep. Default=300.\n"
```

Comment:
Very minor, but the letter `d` for `seconds` is a bit weird. Maybe we could use `-n` for the number of steps and `-s` for the step length?

Reply:
I agree. Awkward for sure. I intended `-d` to be short for `--dt`. I'll see if I can some up with something better. I may just remove the short name for this option.

Reply:
Ah, I see. I think having just `-dt` (and `--dt` if you want both) is probably fine.